docs/migration/00d_Lattice_models_with_v0.5.ipynb

```diff
@@ -0,0 +1,466 @@
+    "### Previously"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "e515413c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Fermionic Operator\n",
+      "register length=4, number terms=10\n",
+      "  (2+0j) * ( +_0 -_2 )\n",
+      "+ (-2+0j) * ( -_0 +_2 )\n",
+      "+ (4+0j) * ( +_0 -_0 )\n",
+      "+ (4+0j) * ( +_2 -_2 )\n",
+      "+ (2+0j) * ( +_1 -_3 )\n",
+      "+ (-2+0j) * ( -_1 +_3 )\n",
+      "+ (4+0j) * ( +_1 -_1 )\n",
+      "+ (4+0j) * ( +_3 -_3 )\n",
+      "+ (3+0j) * ...\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "<ipython-input-1-b4df7055da5b>:1: NatureDeprecationWarning: The qiskit_nature.problems.second_quantization.electronic package is deprecated as of version 0.5.0 and will be removed no sooner than 3 months after the release. Instead use the qiskit_nature.second_q.problems package.\n",
+      "  from qiskit_nature.problems.second_quantization.lattice.lattices import LineLattice\n",
+      "<ipython-input-1-b4df7055da5b>:1: NatureDeprecationWarning: The qiskit_nature.problems.second_quantization.vibrational package is deprecated as of version 0.5.0 and will be removed no sooner than 3 months after the release. Instead use the qiskit_nature.second_q.problems package.\n",
+      "  from qiskit_nature.problems.second_quantization.lattice.lattices import LineLattice\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.problems.second_quantization.lattice.lattices import LineLattice\n",
+    "from qiskit_nature.problems.second_quantization.lattice.models import FermiHubbardModel\n",
+    "\n",
+    "line = LineLattice(2)\n",
+    "fermi = FermiHubbardModel.uniform_parameters(line, 2.0, 4.0, 3.0)\n",
+    "print(fermi.second_q_ops())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e19da0b",
+   "metadata": {},
+   "source": [
+    "### New"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "5cdecf4b",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Fermionic Operator\n",
+      "number spin orbitals=4, number terms=10\n",
+      "  2.0 * ( +_0 -_2 )\n",
+      "+ -2.0 * ( -_0 +_2 )\n",
+      "+ 4.0 * ( +_0 -_0 )\n",
+      "+ 4.0 * ( +_2 -_2 )\n",
+      "+ 2.0 * ( +_1 -_3 )\n",
+      "+ -2.0 * ( -_1 +_3 )\n",
+      "+ 4.0 * ( +_1 -_1 )\n",
+      "+ 4.0 * ( +_3 -_3 )\n",
+      "+ 3.0 * ( +_0 -_0 +_1 -_1 )\n",
+      "+ 3.0 * ( +_2 -_2 +_3 -_3 )\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.second_q.hamiltonians.lattices import LineLattice\n",
+    "from qiskit_nature.second_q.hamiltonians import FermiHubbardModel\n",
+    "\n",
+    "line = LineLattice(2)\n",
+    "fermi = FermiHubbardModel(line.uniform_parameters(2.0, 4.0), 3.0)\n",
+    "print(fermi.second_q_op())"
```

Comment:
So once the Model is constructed the method to get the operator no longer is plural ie has the **s** at the end just second_q_op, not second_q_ops. Should we note this -its a bit more subtle if you don't pay attention

Reply:
Yes, I will do that :+1: 

The reason for that is that all lattice models now implement the `Hamiltonian` interface rather than the `SecondQuantizedProperty` one (which still has `second_q_ops()` because they produce dicts of operators rather than single ones).